scripts/variantstore/variant_annotations_table/ReadMe.md

```diff
@@ -0,0 +1,107 @@
+Creating the Variant Annotations Table
+
+The WDL to run is:
+GvsCreateVAT
+
+And then you’ll want to run:
+GvsValidateVAT
+To check and validate the VAT
+
+To run GvsCreateVAT:
+
+Most of the inputs are constants---like the reference, or a table schema--and dont require any additional work (and paths can be found in the example json). However for the specific data being put in the VAT, three inputs need to be created.
+The first two of these inputs are two files—one of the file/vcf/shards you want to use for the VAT, and one of their indices. These are labelled as inputFileofFileNames and inputFileofIndexFileNames.
+
+You will need to know where all of the GVS extract output files live.
+Example of creating the files of file names and indices:
+! gsutil ls gs://prod-drc-broad/beta-release-99k-v3/beta_99k_*.vcf.gz > inputFileofFileNames.txt
+! gsutil ls gs://prod-drc-broad/beta-release-99k-v3/beta_99k_*.vcf.gz.tbi > inputFileofIndexFileNames.txt
+
+Those two created files will then need to get copied into a GCP bucket that this pipeline will have access to (eg. this bucket: gs://aou-genomics-curation-prod-processing/vat/)
+
+The third input is the ancestry file that Lee has made and it needs to be copied into a GCP bucket that this pipeline will have access to (eg. this bucket: gs://aou-genomics-curation-prod-processing/vat/) for easy access during the workflow. This input has been labelled as the ancestry_file.
+Most of the other files are specific to where the VAT will live, like the project_id and dataset_name and the table_suffix which will name the VAT itself as vat_`table_suffix` as well as a GCP bucket location, the output_path, for the intermediary files and the VAT export in tsv form.
+
+The other inputs can be copied from where I’ve put several files (maybe we should find a better place?!? Since many are in my scratch dir for now—tho anything in my scratch dir is also living in github as it’s things like what BQ schema is needed)
+
+The provided example WDL indicates all of the needed inputs.
+
+
+
+
+
+
+
+Here’s a brief overview of the pipeline itself:
```

Comment:
I think that the task-by-task explanation can be done either here **or** in WDL comments but it doesn't make sense for it to be in two places (too easy to go out-of-sync).  I'd lean toward WDL comments but I don't feel too strongly.